extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/utility.js

```diff
@@ -29,6 +29,9 @@ angular.module('PaperUI.services').factory('util', function($filter, dateTime) {
                     return $filter('date')(date, "dd.MM.yyyy HH:mm:ss");
                 }
             } else if (!item.stateDescription || !item.stateDescription.pattern) {
+                if (item.type && item.type.toUpperCase() == "DIMMER") {
+                    state += "%";
```

Comment:
There was no space there previously but I think it makes sense to have it. Updated.